std/json.d

```diff
@@ -166,7 +165,8 @@ struct JSONValue
     }
 
     /// Value getter/setter for $(D JSON_TYPE.STRING).
-    /// Throws $(D JSONException) for read access if $(D type) is not $(D JSON_TYPE.STRING).
+    /// Throws $(D JSONException) for read access if $(D type) is not
```

Comment:
I think what @yebblies meant is that the word `a` should be removed, not that you should not use the ddoc `Throws` section.